src/coreclr/pal/inc/pal.h

```diff
@@ -3466,6 +3472,55 @@ FORCEINLINE void PAL_ArmInterlockedOperationBarrier()
 #endif
 }
 
+#if defined(HOST_ARM64)
+
+#if defined(LSE_INSTRUCTIONS_ENABLED_BY_DEFAULT)
+
+#define Define_InterlockMethod(RETURN_TYPE, METHOD_DECL, METHOD_INVOC, INTRINSIC_NAME) \
+EXTERN_C PALIMPORT inline RETURN_TYPE PALAPI METHOD_DECL        \
```

Comment:
From the https://github.com/dotnet/runtime/pull/71026#discussion_r90208476, just to make sure that this does generate LSE, I believe this should also be attributed with "lse" and (noinline)